src/legacy/ui/public/field_editor/field_editor.js

```diff
@@ -124,6 +122,7 @@ export class FieldEditorComponent extends PureComponent {
     };
     this.supportedLangs = getSupportedScriptingLanguages();
     this.deprecatedLangs = getDeprecatedScriptingLanguages();
+    this.fieldFormats = npSetup.plugins.data.fieldFormats;
```

Comment:
Not sure that we should add fieldFormats field into FieldEditor class. 

Please use npSetup.plugins.data.fieldFormats directly in places where you need it or create new function out of class like e.g. 
const getFieldFormats = () => npSetup.plugins.data.fieldFormats

Reply:
Done